CHANGELOG.rst

```diff
@@ -185,6 +185,12 @@ Added
 
   Contributed by @nzlosh
 
+* Fix a bug in the pack config loader so that objects covered by an ``patternProperties`` schema
```

Comment:
Unfortunately, this needs moving into the 3.8.0 section.

Reply:
Oops. Thanks. Fixed.